src/dsymbol/tests.d

```diff
@@ -268,6 +268,13 @@ unittest
 	assert(T2.kind == CompletionKind.variadicTmpParam);
 }
 
+unittest
+{
```

Comment:
maybe a comment stating that `this is for testing that internString data is always on the same address for modulecache recursion guard` would make this a bit less random to look at if you just look at this file without knowing this change